src/lib/token-utils.js

```diff
@@ -0,0 +1,41 @@
+import { WEI_PRECISION } from 'src/lib/utils';
+
+/**
+ * Launches a prompt in MetaMask to add a given token as a tracked token, allowing the user to view their balance of
+ * that token at a glance from MetaMask
+ *
+ * @param {string}    address - the address of the token contract
+
+ * @param {string}    symbol - the token's ticker symbol, e.g. 'STLOS'
+ *
+ * @param {string}    image - permalink url of the token's icon
+ *
+ * @param {string}    type - Ethereum standard of the token; default is 'ERC20'
+ *
+ * @param {number}    decimals - the number of decimals constituting the token's precision, default is 18
+ *
+ *
+ * @returns {Promise<boolean>} - true iff the user granted the request to track sTLOS
```

Comment:
```suggestion
 * @returns {Promise<boolean>} - true if the user granted the request to track sTLOS
```

Reply:
`iff` is intentional here, it means 'if and only if'

Reply:
uuuuh I did not know that! Thank you for explaining it!